packages/clerk-js/src/ui/utils/colors/constants.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Shared constants for color utilities
+ * Consolidates all color-related constants to eliminate duplication
+ */
+
+import type { ColorScale } from '@clerk/types';
+
+// Types
+export type ColorShade = 25 | 50 | 100 | 150 | 200 | 300 | 400 | 500 | 600 | 700 | 750 | 800 | 850 | 900 | 950;
+export type ColorShadeKey = keyof ColorScale<any>;
+
+// Core color scale definition
+export const COLOR_SCALE: readonly ColorShade[] = [
+  25, 50, 100, 150, 200, 300, 400, 500, 600, 700, 750, 800, 850, 900, 950,
+] as const;
+
+// Shade groupings for scale generation
+export const LIGHT_SHADES: ColorShadeKey[] = ['25', '50', '100', '150', '200', '300', '400'];
```

Comment:
this is the third time I've seen this code, if we're maintaining two separate implementations, one for legacy and one for modern, let's make it clear at the top of the file which is which